pkg/conthandler/container_main_handler.go

```diff
@@ -120,7 +120,7 @@ func (ch *ContainerHandler) startTimer(watchedContainer watchedContainerData, co
 }
 
 func createTicker() *time.Ticker {
-	return time.NewTicker(time.Duration(config.GetConfigurationConfigContext().GetUpdateDataPeriod()) * time.Second)
+	return time.NewTicker(time.Duration(config.GetConfigurationConfigContext().GetUpdateDataPeriod()))
```

Comment:
do you still need to convert to time.duration here?

Reply:
forgot to push this file